flang/test/Lower/OpenMP/single.f90

```diff
@@ -0,0 +1,131 @@
+!RUN: %flang_fc1 -emit-hlfir -fopenmp %s -o - | FileCheck %s
+!RUN: bbc -emit-hlfir -fopenmp %s -o - | FileCheck %s
+
+!===============================================================================
+! Single construct
+!===============================================================================
+
+!CHECK-LABEL: func @_QPomp_single
+!CHECK-SAME: (%[[x:.*]]: !fir.ref<i32> {fir.bindc_name = "x"})
```

Comment:
Nit: Change `x` to a capital `X`, here and elsewhere.

Reply:
Done